spec/unit_tests/moab/storage_object_validator_spec.rb

```diff
@@ -7,9 +7,15 @@
   let(:storage_obj_validator) { described_class.new(storage_obj) }
   let(:error_list) { storage_obj_validator.validation_errors }
 
-  before { described_class::IMPLICIT_DIRS = ['.', '..', '.keep'].freeze }
+  before do
+    # avoid warning: already initialized constant
+    silence_warnings { described_class::IMPLICIT_DIRS = ['.', '..', '.keep'].freeze }
```

Comment:
the instruction was to use RSpec `stub_const` but it only works for string arguments.  I couldn't come up with a better solution than this.